spec/lib/api/v3/users/user_representer_spec.rb

```diff
@@ -55,10 +57,38 @@
       it { is_expected.to have_json_path('avatar') }
     end
 
+    describe 'status' do
+      it 'contains the name of the account status' do
+        expect(parse_json(subject, 'status')).to eql 'active'
+      end
+    end
+
     describe '_links' do
       it 'should link to self' do
         expect(subject).to have_json_path('_links/self/href')
       end
+
+      context 'when regular current_user' do
+        it 'should have no lock-related links' do
+          expect(subject).to_not have_json_path('_links/lock/href')
+          expect(subject).to_not have_json_path('_links/unlock/href')
+        end
+      end
+
+      context 'when current_user is admin' do
+        let(:current_user) { FactoryGirl.create(:admin) }
```

Comment:
Whenever possible, I would like to have build_stubbed/build used instead of create. At least the tests of the higher layers (view, controller) should strive for it. Using it speeds up the tests. It might not matter for one test but doing that on a lot of them does.